src/vivarium_census_prl_synth_pop/results_processing/formatter.py

```diff
@@ -92,6 +92,22 @@ def format_age(data: pd.DataFrame) -> pd.Series:
     return data["age"].astype(int)
 
 
+def format_copy_age(data: pd.DataFrame) -> pd.Series:
+    no_copy_idx = data.index[data["copy_age"].isna()]
```

Comment:
Can't you just do the same thing as format_age? `return data["copy_age"].astype(int)`. This way you don't need to convert to string and split.

Reply:
See other comment.